runtime/Stdlib_Error.resi

```diff
@@ -155,8 +155,28 @@ if 5 > 10 {
 }
 ```
 */
+@deprecated(
+  "`raise` has been renamed to `throw` to align with JavaScript vocabulary. Please use `throw`"
+)
 external raise: t => 'a = "%raise"
 
+/**
+Throw and exception provided `Error.t`, which will stop execution.
+
+## Examples
+
+```rescript
+let error = Error.make("Everything is upside down.")
+
+if 5 > 10 {
+  error->Error.throw
```

Comment:
I'd prefer not to use pipe in this example to make it more familiar to JS users:

```suggestion
 throw(error)
```